.github/workflows/build_release_assets.yml

```diff
@@ -64,12 +65,44 @@ jobs:
           # Get enough commits to run `ggshield secret scan commit-range` on ourselves
           fetch-depth: 10
 
-      - name: Set up Python 3.10
+      - name: Set up Python 3.10 (Windows 1/2)
+        if: matrix.os == 'windows-2022'
         uses: actions/setup-python@v5
-        if: "!startsWith(matrix.os, 'macos-')"
         with:
           python-version: '3.10'
 
+      - name: Set up Python 3.10 (Windows 2/2)
+        if: matrix.os == 'windows-2022'
+        shell: bash
+        run: |
+          echo PYTHON_CMD=python >> $GITHUB_ENV
+
+      - name: Install Linux specific dependencies
+        if: matrix.os == 'ubuntu-22.04'
+        run: |
+          # Install necessary packages
+          yum install -y \
+            python3.9 \
+            git-core \
+            findutils
+
+          echo PYTHON_CMD=/usr/bin/python3.9 >> $GITHUB_ENV
+
+          # Install NFPM
+          NFPM_VERSION=2.36.1
+          NFPM_CHECKSUM=9f8effa24bc6033b509611dbe68839542a63e825525b195672298c369051ef0b
+
+          scripts/download \
+            https://github.com/goreleaser/nfpm/releases/download/v${NFPM_VERSION}/nfpm_${NFPM_VERSION}_Linux_x86_64.tar.gz \
+            nfpm.tar.gz \
+            $NFPM_CHECKSUM
+
+          tar xf nfpm.tar.gz nfpm
+          cp nfpm /usr/local/bin
+
+          # Avoid "dubious permission" git error
+          git config --global --add safe.directory '*'
```

Comment:
Why do we need this? We don't use git to build the binary, or am I missing something?

Reply:
Good question, we probably don't need it anymore. It was required before I replaced the `--git-version` arg with `--suffix`. Going to check if we can do without it.

Reply:
You were right: not necessary anymore :+1:.